node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+import { readFile, writeFile } from 'node:fs/promises';
+async function copy() {
+  try {
+    const name = process.argv.slice(2);
+    const newFile = name.slice(1);
+    const fileTxt = await readFile(`${name}`, 'utf8');
```

Comment:
It would be best to do the readFile without an encoding. With it, you can only copy text files. If you leave it off you'll be able to copy images, etc., as well.